packages/next-dashboard/src/components/charts/PageChart.js

```diff
@@ -2,14 +2,12 @@
 import React from 'react';
 /*:: import * as R from 'react'; */
 
-type Props = {
-  children: R.Node,
-};
-
-const PageChart = ({ children }: Props) => (
-  <div className="page-chart">
-    <div className="page-chart-content">{children}</div>
-  </div>
-);
+function PageChart({ children }: { children?: R.Node }): R.Node {
```

Comment:
Why mark the `children` prop as optional? Surely it doesn't make much sense to render a children-less PageChart(?)

Reply:
Flow complained when PageChart didn't have an optional children prop, when React.Fragment had in this line:
`const Wrapper = pageChart ? PageChart : React.Fragment;`

Reply:
Huh, curious. Okay then, I guess